src/app/DIAmeterApplication.h

```diff
@@ -0,0 +1,197 @@
+/**
+ * DIAmeterApplication.h
+ * DATE: June 15, 2021
+ * AUTHOR: Yang Lu
+ * DESCRIPTION: The DIAmeter module for DIA search.
+ **************************************************************************/
+
+#ifndef DIAMETERAPPLICATION_H
+#define DIAMETERAPPLICATION_H
+
+#include "CruxApplication.h"
+#include "TideMatchSet.h"
+#include "TideSearchApplication.h"
+
+#include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <gflags/gflags.h>
+#include "peptides.pb.h"
+#include "spectrum.pb.h"
+#include "tide/theoretical_peak_set.h"
+#include "tide/max_mz.h"
+
+using namespace std;
+
+// It is modified from TideSearchApplication ScSortByMz with following differences:
+// By default the mz tolerane would be half of the isolation window
+// In case the isolation window is 0 which should cause a fatal error elsewhere
```

Comment:
Is each line its own difference? It it a little hard to tell .
I feel like we are missing the second half of this. If the isolation window is zero - then what?